### DISCUSSION TITLE
Dev services mongodb version

### DISCUSSION THREAD

**Discussion by ennishol** [quarkusio/quarkus#37517] [Community]
With the latest quarkus, dev services start  docker.io/mongo:4.4 container. Is there a way to change the version used? I'd like to use MongoDB 6
Thanks!

**Comment by maxandersen** [comment]
https://quarkus.io/guides/dev-services#quarkus-mongodb-config-group-dev-services-build-time-config_quarkus.mongodb.devservices.image-name

**Reply by ennishol** [reply]
Thanks! I need new glasses....
